README.md

```diff
@@ -1,17 +1,31 @@
 # Memo Index
 
+### Install deps
+```bash
+dnf install -y golang,bzr
```

Comment:
Instead of system specific instructions, can just link to Go docs: https://go.dev/doc/install